boards/nordic/nrf54l15dk/nrf54l15dk_nrf54l05_cpuapp.yaml

```
@@ -0,0 +1,24 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: nrf54l15dk/nrf54l05/cpuapp
+name: nRF54l15-DK-nRF54l05-Application
```

```suggestion
name: nRF54L15-DK-nRF54L05-Application
```
Applies also to L10 and L15.